test/requests/artefact_request_test.rb

```diff
@@ -142,6 +142,15 @@ def bearer_token_for_user_without_permission
     end
   end
 
+  it 'should return a downtime message if downtime is scheduled' do
+    downtime = FactoryGirl.create(:downtime, artefact: FactoryGirl.create(:live_artefact_with_edition))
+
+    get "/#{downtime.artefact.slug}.json"
+
+    assert_equal 200, last_response.status
+    assert_equal downtime.message, JSON.parse(last_response.body)['details']['downtime']['message']
+  end
```

Comment:
Would be good to include a test for when a `Downtime` exists, but is not current (ie `publicise?` returns false)